package/patterns-yast.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Apr 11 14:19:26 CEST 2019 - Antoine Ginies <aginies@suse.com>
+
+- remove requires on old package yast2-branding-openSUSE
```

Comment:
Just to make sure no bot complains about the lack of bugzilla ids, I would add "(related to boo#1109310)" here. For symmetry with https://github.com/yast/patterns-yast/pull/22/files

Reply:
ok i re-add the BSC number